ui/src/components/MonitorComp/index.tsx

```diff
@@ -43,40 +41,40 @@ export default function MonitorComp({
   const { data: allMetrics } = useRequest(getAllMetrics, {
     defaultParams: [queryScope],
   });
-  const [visible, setVisible] = useState(false);
-  const [modalProps, setModalProps] = useState<LineGraphProps>({});
-  const Title = ({
-    metrics,
-    name,
-  }: {
-    metrics: MetricType[];
-    name: string;
-  }) => {
-    return (
-      <div>
-        {name}
-        <Tooltip
-          title={
-            <ul>
-              {metrics.map((metric, idx) => (
-                <li key={idx}>
-                  {metric.name}:{metric.description}
-                </li>
-              ))}
-            </ul>
-          }
-        >
-          <QuestionCircleOutlined
-            style={{
-              color: 'rgba(0, 0, 0, 0.45)',
-              cursor: 'help',
-              marginLeft: '4px',
-            }}
-          />
-        </Tooltip>
-      </div>
-    );
-  };
+  // const [visible, setVisible] = useState(false);
```

Comment:
Why comment these lines? Are they unused?

Reply:
Yes, none of them are used, but I think they might be used in the future.

Reply:
OK, I see